src/exchange/account/home/account-home.component.js

```diff
@@ -82,10 +86,17 @@
     }
 
     fetchCanUserSubscribeToOfficeAttach() {
-      return this.officeAttach
-        .retrievingIfUserAlreadyHasSubscribed(this.$routerParams.productId)
-        .then((userHasAlreadySubscribed) => {
-          this.userCanSubscribeToOfficeAttach = !userHasAlreadySubscribed;
+      return this.OvhApiMe.v6().get().$promise
+        .then(({ ovhSubsidiary }) => {
+          if (['CA'].includes(ovhSubsidiary)) {
+            return this.$q.when(true);
```

Comment:
```suggestion
            return this.$q.when(false);
```

User can't subscribe if they are from CA